### ISSUE TITLE
If a controller has an action method named "SignIn" then the Generate-R4MVC command does not work

### ISSUE THREAD

**Issue by Shubham-Saini** [T4MVC/R4MVC#128]
If a controller has an action method named "SignIn", then that method is not converted by this tool.
```
public class DefaultController : Controller
    {
        public IActionResult SignIn()
        {
            return null;
        }
    }
```
The method does not become virtual and the generated partial class has no reference to this method, however an interesting thing to point here is that, if I change the name of this method or even the case of the word "SignIn" it works as expected.
This test was done in .NET Core 2.0

**Comment by MohammadMokhtarii** [comment]
have same Issue
Do You Found Out The Solution/??
@Shubham-Saini 
@BrianSchroer 
@davidebbo

**Comment by Shubham-Saini** [comment]
Yes I did, as it turns out there is already a method named "SignIn". I
can't remember in which class exactly, which causes this behavior. A simple
fix was just to change the method name. As I recall I changed it to
"Signin" and it worked perfectly after that.

**Comment by btihanyi** [comment]
The problem is that the _SingIn_ name is [used by](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.controllerbase.signin) the `ControllerBase` ancestor class. The R4MVC tool [skips all the action methods](https://github.com/T4MVC/R4MVC/blob/develop/src/R4Mvc.Tools/Extensions/SyntaxNodeHelpers.cs#L88) which names are the same (case-sensitively) as one of their ancestor classes. I find this unnecessary, because they are all marked with the `[NonAction]` attribute, so they would be skipped anyway in the generation process (at least for MVC controller actions). So yeah, the current workaround is to change the action name to lowercase.
